.travis.yml

```diff
@@ -0,0 +1,31 @@
+language: php
+php:
+- '7.0'
```

Comment:
Differences seem to largely esoteric.set it to 7.0, i doubt we'll hit rough ground.